src/elements/types/ExportSpecifier.js

```diff
@@ -0,0 +1,43 @@
+import ModuleSpecifier from '../ModuleSpecifier';
+
+// tests are in ExportNamedDeclaration
+
+export default class ExportSpecifier extends ModuleSpecifier {
+    constructor(childNodes) {
+        super('ExportSpecifier', childNodes);
+    }
+
+    _acceptChildren(children) {
+        console.log(children._elements.map(function(e) {
+            return e._type;
+        }));
+
+        let local;
+        console.log(children._currentElement);
+        console.log(children.isNode('Identifier'));
+        console.log(children.isToken('Identifier'));
```

Comment:
Well, `import {x}` is a shortcut for `import {x as x}`, so if I understand correctly we have:

`import {x}` -> `Token('Keyword', 'import') Token('Punctuator', '{') Identifier([Token('Identifier', 'x')]) Token('Punctuator', '}')`
`import {x as y}` -> `Token('Keyword', 'import') Token('Punctuator', '{') Token('Identifier', 'x') Token('Identifier', 'as') Identifier([Token('Identifier', 'y')]) Token('Punctuator', '}')`

So depending on the input you made decision: whether this is shortcut, or naming an export or even a default (`import {default as z}`).